custmer.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&family=Open+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Fashion-Custmer page</title>
+</head>
+<body>
+    <div class="navbar cs">
+        <h2 class="brandName">Fashion</h2>
+        <div class="search-container">
+         <form >
+           <input type="text" placeholder="Search.." name="search">
+           <button type="submit"><i class="fa fa-search"></i></button>
+         </form>
+        </div>
+        <button id="custmerbtn">Custmer</button>
+        <button id="sellerbtn">Seller</button>
+        <button id="homebtn"><i class="fa fa-home "></i></button>
+    </div>
+
+    <div class="navmenu">
+        <a href="#">Home</a>
```

Comment:
You don't need `home` link here since it's already on the top right of the page. choose one of them.